pnpbridge/src/adapters/src/impinj_reader_r700/support_files/config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../../../pnpbridge/src/pnpbridge_config_schema.json",
+  "pnp_bridge_connection_parameters": {
+    "connection_type" : "dps",
+    "root_interface_model_id": "dtmi:impinj:R700;1",
+    "auth_parameters": {
+        "auth_type": "symmetric_key",
+        "symmetric_key": "lZKGVYbHYRcaJkeJt4COQ6RxtBb61IHmO783Gx13lks="
+    },
+    "dps_parameters": {
+        "global_prov_uri": "global.azure-devices-provisioning.net",
+        "id_scope": "0ne001C686E",
+        "device_id": "kwenner-R700"
```

Comment:
Insert placeholders like "<insert-id-scope>" to keep sensitive information safe.